duckduckgo/iOS
# Switch to using search page filters

Reviewer: Caine
Asana: https://app.asana.com/0/361428290920652/385354519462944

**Description**:
In app and search page filters are fighting. This PR removes the filters in settings and protects the search filters from being blown away when the cache is cleared

**Steps to test this PR**:
TEST 1:
1. Open settings
2. Note that filter options are no longer there

TEST 2:
1. Perform a search and switch off the safe search option 
2. Open tabs, press "forget" to clear the cache
3. Perform a new search and check that the safe search option is still off

###### Reviewer Checklist:

###### PR DRI Checklist:
